Translations/uk.po

```diff
@@ -1579,120 +1579,128 @@ msgstr "Пауза"
 
 #: Source/help.cpp:26
 msgid "$Keyboard Shortcuts:"
-msgstr ""
+msgstr "$Гарячі Клавіші:"
 
 #: Source/help.cpp:27
 msgid "F1:    Open Help Screen"
-msgstr ""
+msgstr "F1:    Відкрити вікно Допомоги"
 
 #: Source/help.cpp:28
 msgid "Esc:   Display Main Menu"
-msgstr ""
+msgstr "Esc:   Вікрити Головне Меню"
 
 #: Source/help.cpp:29
 msgid "Tab:   Display Auto-map"
-msgstr ""
+msgstr "Tab:   Відкрити Автокарту"
 
 #: Source/help.cpp:30
 msgid "Space: Hide all info screens"
-msgstr ""
+msgstr "Пробіл: Сховати всі вікна"
 
 #: Source/help.cpp:31
 msgid "S: Open Speedbook"
-msgstr ""
+msgstr "S: Відкрити Швидку Книгу Чар"
 
 #: Source/help.cpp:32
 msgid "B: Open Spellbook"
-msgstr ""
+msgstr "B: Відкрити Книгу Чар"
 
 #: Source/help.cpp:33
 msgid "I: Open Inventory screen"
-msgstr ""
+msgstr "I: Відкрити вікно Інвентаря"
 
 #: Source/help.cpp:34
 msgid "C: Open Character screen"
-msgstr ""
+msgstr "C: Вікрити вікно Героя"
 
 #: Source/help.cpp:35
 msgid "Q: Open Quest log"
-msgstr ""
+msgstr "Q: Вікрити Журнал квестів"
 
 #: Source/help.cpp:36
 msgid "F: Reduce screen brightness"
-msgstr ""
+msgstr "F: Зменшити яскравість екрану"
 
 #: Source/help.cpp:37
 msgid "G: Increase screen brightness"
-msgstr ""
+msgstr "G: Збільшити яскравість екрану"
 
 #: Source/help.cpp:38
 msgid "Z: Zoom Game Screen"
-msgstr ""
+msgstr "Z: Збільшити/зменшити гру"
 
 #: Source/help.cpp:39
 msgid "+ / -: Zoom Automap"
-msgstr ""
+msgstr "+ / -: Збільшити/зменшити Автокарту"
 
 #: Source/help.cpp:40
 msgid "1 - 8: Use Belt item"
-msgstr ""
+msgstr "1 - 8: Використати предмет з Поясу"
 
 #: Source/help.cpp:41
 msgid "F5, F6, F7, F8:     Set hotkey for skill or spell"
-msgstr ""
+msgstr "F5, F6, F7, F8:     Встановити гарячу клавішу для чар або таланту"
```

Comment:
How are we translating skill? Talant? 

I think навичка is better, but not a big problem.

Reply:
Disagree on "Навичка", since it's a habit, not a skill
"Талант" is closer to game's meaning of "Skill" since it's an innate skill of a character that they can use immediately.